### ISSUE TITLE
some of the differences with offical repo

### ISSUE THREAD

**Issue by twmht** [ZJCV/overhaul#4]
Hi,

I found out some of the differnces between yours and offical's. 

For example, you divide the number of negative point in the loss (https://github.com/ZJCV/overhaul/blob/master/ofd/criterion/ofd_loss.py#L40), why you did this?

another one is that you resize the image to 224x224 (https://github.com/ZJCV/overhaul/blob/master/configs/ofd/resnet/ofd_2_0_r152_pret_r18_c100_224_e100_sgd_mslr.yaml#L26), why you did this?

**Comment by zjykzj** [comment]
> 

Hi @twmht ，

for the first question, it is debugged during the actual test. The loss is so big making the training unexpected, dividing the number of mask can make training more steedy

for second question, it's just for use, no obvious tricks. You can try other way

**Comment by twmht** [comment]
@zjykzj 

> The loss is so big making the training unexpected

Yup. I also found out this, the official's division is very weird. but have you ever tested the offical repo? if this is a problem, then official's may not work as expected.

**Comment by zjykzj** [comment]
> @zjykzj
> 
> > The loss is so big making the training unexpected
> 
> Yup. I also found out this, the official's division is very weird. but have you ever tested the offical repo? if this is a problem, then official's may not work as expected.

No tried, just checked some implementations. How about you ? have your try that ?

**Comment by twmht** [comment]
I have not tried that. and the paper did not mention about that.

I am going to try and see if any unstable happens.

**Comment by twmht** [comment]
by the way, there are some better fix for this repo.

For example,

https://github.com/ZJCV/overhaul/blob/master/ofd/distill/ofd_distiller.py#L24

I think adding `bias=False` is better since you have bn after that.

**Comment by zjykzj** [comment]
> 

Nice, give a PR please

**Comment by zjykzj** [comment]
> by the way, there are some better fix for this repo.
> 
> For example,
> 
> https://github.com/ZJCV/overhaul/blob/master/ofd/distill/ofd_distiller.py#L24
> 
> I think adding `bias=False` is better since you have bn after that.

Nice, give a PR please
